2018/python/test_file_generator.py

```diff
@@ -2866,11 +2874,16 @@ def q604(self):
 		path = self.quality_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.state = lar.state.map(lambda x: random.choice(list(self.crosswalk_data.stateCode)))
+		lar.state = lar.state.map(lambda x: self.state_abbrev[random.choice(list(self.crosswalk_data.stateCode))])
+		print(lar.state)
 		#this implemenation sets all state codes to the same code and uses that to make a county list 
```

Comment:
Is there another implementation that is used based on different configurations?

Reply:
Hey David, can you elaborate? I am not sure what you mean by your question.

Reply:
On line 2879 there is a comment stating `this implementation sets all state codes...`. I was unsure if there were other configurations I should be reviewing.

Reply:
The only configuration used here is the geographic configuration. That configuration was reviewed previously, and so there is no need to review that here.

Reply:
Can you change the code comment to reflect this clarification?

Reply:
Yes, the code comment has been changed to remove ambiguity.